src/cephadm/cephadm

```diff
@@ -1012,7 +1012,16 @@ def get_config_and_both_keyrings():
             with open(args.config_and_keyrings, 'r') as f:
                 j = f.read()
         d = json.loads(j)
-        return (d.get('config'), d.get('keyring'), d.get('crash_keyring'))
+        config = keyring = crash_keyring = None
+        if d.get('config'):
+            config = d.pop('config')
+        if d.get('keyring'):
+            keyring = d.pop('keyring')
+        if d.get('crash_keyring'):
```

Comment:
Remove this block in favor of the logic below.... the multiline code suggestions in github didn't work right :frowning_face:

Reply:
no it didn't. Thanks github.

Reply:
indeed, thanks github :rofl: